static/index.html

```diff
@@ -398,6 +396,7 @@ <h2>Measurements</h2>
         if (window.lint) {
           var doc = CodeMirror.Doc(text);
           var errors = window.lint(text);
+          console.log(errors)
```

Comment:
Needed for debugging on the deployed version or should that be removed?

Reply:
It should be removed.